File: src/web/stackforge-web/src/app/features/mentorship/models/GetSentMentorshipRequest/get-sent-mentorship-request.response.ts
export interface GetSentMentorshipRequestResponse {
  mentorshipRequestId: string;
  mentorId: string;
  mentorName: string;
  stackId: string;
  stackName: string;
  goal: string;
  status: string;
  createdAt: string;
  decidedAt: string | null;
}

File: src/api/src/StackForge.Application/MentorshipContext/UseCases/GetSentMentorshipRequest/GetSentMentorshipRequestResponse.cs
namespace StackForge.Application.MentorshipContext.UseCases.GetSentMentorshipRequest;

public sealed record GetSentMentorshipRequestResponse(
    Guid MentorshipRequestId,
    Guid MentorId,
    string MentorName,
    Guid StackId,
    string StackName,
    string Goal,
    string Status,
    DateTimeOffset CreatedAt,
    DateTimeOffset? DecidedAt);
